src/Symfony/Component/Routing/Loader/AnnotationClassLoader.php

```diff
@@ -219,42 +313,34 @@ protected function getGlobals(\ReflectionClass $class)
             'name' => '',
         );
 
-        if ($annot = $this->reader->getClassAnnotation($class, $this->routeAnnotationClass)) {
-            if (null !== $annot->getName()) {
-                $globals['name'] = $annot->getName();
-            }
-
-            if (null !== $annot->getPath()) {
-                $globals['path'] = $annot->getPath();
-            }
-
-            if (null !== $annot->getRequirements()) {
```

Comment:
Please avoid reorganizing this code. It makes merging branches harder (and again, for no actual benefit)